tests/tests.c

```diff
@@ -1065,24 +1069,24 @@ bool sirtest_levelssanity(void) {
         }
 
         _sir_eqland(pass, _sir_validlevels(levels));
-        (void)printf(INDENT_ITEM WHITE("(%zu/%zu): random valid (count: %"PRIu32", levels:"
-                                 " %04"PRIx16) ")\n", n + 1, iterations, rand_count, levels);
+        (void)printf(INDENT_ITEM SIR_WHITE("(%zu/%zu): random valid (count: %"PRIu32", levels:"
+                                 " %04"PRIx16) ")" SIR_EOL, n + 1, iterations, rand_count, levels);
     }
-    PASSFAIL_MSG(pass, "\t--- random bitmask of valid levels: %s ---\n\n", PRN_PASS(pass));
+    PASSFAIL_MSG(pass, "\t--- random bitmask of valid levels: %s ---" SIR_EOL SIR_EOL, PRN_PASS(pass));
 
-    TEST_MSG_0(WHITEB("--- invalid values ---"));
+    TEST_MSG_0(SIR_WHITEB("--- invalid values ---"));
 
     /* greater than SIRL_ALL. */
     sir_levels invalid = (0xffffu & ~SIRL_ALL);
```

Comment:
We use an upper case `U` suffix down below a few lines... one of these is wrong. I don't remember which one we use more of but they should all be the same...

Reply:
It should be upper.  I'll fix.